.github/workflows/graphql.yaml

```diff
@@ -6,6 +6,8 @@ jobs:
     name: GraphQL Inspector
     runs-on: ubuntu-latest
     timeout-minutes: 5
+    env:
+      ACTIONS_STEP_DEBUG: true
```

Comment:
Does this work? I only know setting it via secret

Reply:
only as a secret, I put it there by mistake